intro/setup/index.html

```diff
@@ -0,0 +1,349 @@
+---
+layout: default
+title: Basic window setup
+---
+
+<body>
+    <h1>Basic window setup for pygame</h1>
+    <p>In this article, we will go over the structure to open and keep the window running in pygame.</p>
+
+    <h2>Table of contents</h2>
+    <p>
+        <ol id="toc">
+            <!--
+            To be filled by javascript
+            -->
+        </ol>
+    </p>
+
+    <h2><a id="what-is-pygame" class="js"></a>What is pygame?</h2>
+    <p>
+        Before we get into the nitty gritty parts of it, let us first understand what pygame is.
+        pygame is a free and open source library for the Python programming language, which allows 
+        us to build multimedia applications such as video games.
+    
+    </p>
+    <p>
+        pygame provides the very basic utility to draw and manipulate pixels 
+        on to the screen, much like SDL2, but with extended utility in various forms 
+        that makes it easier to use. pygame gives you direct control over how you 
```

Comment:
Capitalize pygame?

Reply:
I believe we agreed on `pygame` being the way we refer to it throughout the article

Reply:
This particular one, I meant at the beginning of the sentence at the end of this selection